app/components/HeartButton.tsx

```diff
@@ -32,17 +32,18 @@ const HeartButton: React.FC<HeartButtonProps> = ({
       "
     >
       <AiOutlineHeart
-        size={28}
-        className="
-          fill-white
-          absolute
-          -top-[2px]
-          -right-[2px]
-        "
+        className={`absolute ${
+          size === "large"
+            ? "carousel-heart-outline-large"
+            : "carousel-heart-outline-small"
+        } ${hasFavorited ? "fill-transparent" : "fill-white"}`}
```

Comment:
Use `${hasFavorited ? "fill-accent-500" : "fill-white"}`  instead? 

when so when it's fav, ppl can see the lil heart